services/frontend/src/components/tasks/task-cell.js

```diff
@@ -45,18 +51,25 @@ class TaskCell extends React.Component {
 
     render() {
         return (
-            <div
-                className={classNames(style.taskCell, { [style.selected]: this.props.selected })}
-                onClick={this.select}
-            >
-                <input type="checkbox" value={_.get(this.state, 'task.complete', false)} onChange={this.handleChange} />
-                <span>{_.get(this.state, 'task.title')}</span>
-                {!_.isEmpty(_.get(this.state, 'task.description')) && (
-                    <React.Fragment>
-                        <br />
-                        <span style={{ color: 'gray' }}>{this.state.task.description}</span>
-                    </React.Fragment>
-                )}
+            <div className={styles.taskCell}>
+                <ListItem selected={this.props.selected} onClick={this.select}>
+                    <ListItemIcon>
+                        <Checkbox
+                            edge="start"
+                            checked={_.get(this.state, 'task.complete', false)}
+                            onChange={this.handleChange}
+                        />
+                    </ListItemIcon>
+                    <ListItemText
+                        primary={_.get(this.state, 'task.title')}
+                        secondary={_.get(this.state, 'task.description')}
+                    />
+                    <ListItemSecondaryAction className={styles.actions}>
+                        <IconButton edge="end" onClick={this.deleted}>
+                            <DeleteIcon color="error" />
+                        </IconButton>
+                    </ListItemSecondaryAction>
+                </ListItem>
```

Comment:
Inspired by [this example](https://material-ui.com/components/lists/#interactive)